### ISSUE TITLE
Buttons do not display focus outline in Firefox

### ISSUE THREAD

**Issue by bjankord** [cerner/terra-button#10]
### Description of Issue

The focus outline is not displayed in Firefox on buttons when they receive focus.

In both of the screenshots below, the Default button has focus. However, the focus outline is not displayed in Firefox.

Screenshot from Firefox
<img width="769" alt="screen shot 2016-10-19 at 12 53 18 pm" src="https://cloud.githubusercontent.com/assets/633148/19530860/3bd816d4-95fb-11e6-8c3c-e166477c97c5.png">

Screenshot from Chrome showing expected focus outline behavior
<img width="764" alt="screen shot 2016-10-19 at 12 53 38 pm" src="https://cloud.githubusercontent.com/assets/633148/19530886/547ee1fe-95fb-11e6-9292-f7023af0b473.png">
### System Configuration

Mac OS X El Capitan v.10.11.5
#### Component Version

v0.1.0
#### Browser and Version

Firefox 49.0.1
### Steps to Reproduce the Issue
1. Tab to button to focus element
### Expected Outcomes
- Button displays some focus outline

**Comment by Nerdenator** [comment]
I could *possibly* carve out some free time in the next month to address this, not sure if it's in your timeframe though.

**Comment by bjankord** [comment]
That sounds good @Nerdenator, if I get some time, I may pick this up as well.

**Comment by Nerdenator** [comment]
going to give it a try

**Comment by gneatgeek** [comment]
@Nerdenator, let us know if you have any questions or need any help with this issue.

**Comment by bjankord** [comment]
Here is the tweet that prompted me to log this issue: https://twitter.com/zachleat/status/788751828536008705

The fix I'd recommend is to use :-moz-focusring to specify own FF-only outline.

```
// Fixes https://github.com/cerner/terra-button/issues/10
.terra-Button:-moz-focusring {
  outline: 2px solid -moz-mac-focusring;
  -moz-outline-radius: 0.45em;
}
```

**Comment by Nerdenator** [comment]
@bjankord Just implemented; pushing to my fork on commit [c95b091](https://github.com/Nerdenator/terra-button/commit/c95b0910c279ad0e5ddf4de725a8814782ba570f).

**Comment by gneatgeek** [comment]
@Nerdenator, when you're ready, please issue a pull request :)

**Comment by bjankord** [comment]
Fixed via #12
